### PR TITLE
Concat also the other results

### PR DESCRIPTION
**Pull request by LukasReschke**
Otherwise this will ignore the two last checks :speak_no_evil:

@MorrisJobke What we talked about :wink:

### CODE REVIEW

**Comment by MorrisJobke** [comment]
:+1:

File: settings/js/admin.js [hunk 1]
```diff
@@ -172,8 +172,8 @@ $(document).ready(function(){
 		OC.SetupChecks.checkWellKnownUrl('/.well-known/carddav/', oc_defaults.docPlaceholderUrl),
 		OC.SetupChecks.checkSetup(),
 		OC.SetupChecks.checkGeneric()
-	).then(function(check1, check2, check3) {
-		var messages = [].concat(check1, check2, check3);
+	).then(function(check1, check2, check3, check4, check5) {
+		var messages = [].concat(check1, check2, check3, check4, check5);
```
Comment (MorrisJobke) [inline settings/js/admin.js]: Why not use `arguments`?

Comment (LukasReschke) [inline settings/js/admin.js]: Je ne parle pas Javascript :wink:

**Comment by DeepDiver1975** [comment]
@karlitschek this should fix your issue regarding the missing warning about htaccess and data dir

**Comment by Xenopathic** [comment]
:+1:

**Comment by karlitschek** [comment]
:+1:  Thanks

**Pull request merged**